tests/test_symbolic_data.py

```diff
@@ -58,7 +58,7 @@ def test_derivatives_space(derivative, dimension, order):
     # Establish native sympy derivative expression
     width = int(order / 2)
     indices = [(dimension + i * h) for i in range(-width, width + 1)]
-    s_expr = as_finite_diff(u.diff(dimension), indices)
+    s_expr = -as_finite_diff(u.diff(dimension), indices)
```

Comment:
For some reason I haven't found yet (as said in the original message) `finite_diff_weights` and `as_finite_diff` return the opposute sign.